packages/lesmis/src/components/SearchBar/SearchBar.js

```diff
@@ -180,51 +177,53 @@ export const SearchBar = ({ linkType, className }) => {
   const showNoResults = popupOpen && inputValue && focused && !dirty;
 
   return (
-    <SearchContainer className={className}>
```

Comment:
All that's happening here is the search component getting wrapped in the Fade component. No other change